packages/ui-extensions/docs/surfaces/point-of-sale/reference/helpers/returnExtensionTarget.ts

```diff
@@ -0,0 +1,19 @@
+export enum ExtensionTargetType {
+  PosHomeModalRender = 'pos.home.modal.render',
+  PosHomeTileRender = 'pos.home.tile.render',
+  PosPurchasePostActionMenuItemRender = 'pos.purchase.post.action.menu-item.render',
+  PosPurchasePostActionRender = 'pos.purchase.post.action.render',
+}
+
+export function returnExtensionTarget(type: ExtensionTargetType): string {
+  switch (type) {
+    case ExtensionTargetType.PosHomeModalRender:
+      return 'pos.home.modal.render';
+    case ExtensionTargetType.PosHomeTileRender:
+      return 'pos.home.tile.render';
+    case ExtensionTargetType.PosPurchasePostActionMenuItemRender:
+      return 'pos.purchase.post.action.menu-item.render';
+    case ExtensionTargetType.PosPurchasePostActionRender:
+      return 'pos.purchase.post.action.render';
+  }
+}
```

Comment:
I believe .toString() should work on enums like this.

`ExtensionTargetType.PosHomeModalRender.toString()`

So we shouldn't need the `returnExtensionTarget` function at all